addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -17,6 +17,11 @@
     ('3', 'Excellent')
 ]
 
+JOB_POSTING_SERVICES_MAILS = {
```

Comment:
To me I think it would feel more natural to have this dictionary defined in the method that uses it, since it's only used in one place. That way it's also more clear what it's for. Analogous to how you defined job_offer_parsers.

Also I think the name job_posting_services_mails sounds like the mail addresses will be the values of the dictionary, so I would either rename it to job_posting_services or maybe switch the keys and values around, so the email addresses are the values and the name of the service being the key, again consistent with how you implemented job_offer_parsers :)